iron-overlay-manager.html

```diff
@@ -45,7 +45,11 @@
     }
 
     function currentOverlay() {
-      return overlays[overlays.length-1];
+      var i = overlays.length - 1;
```

Comment:
@samccone thanks, I can read the for loop :) I just want a comment so that we know why we wrote this in the first place (i.e. whatever @cdata wrote in the description)